tikv-control.md

```diff
@@ -139,17 +139,65 @@ This section describes the subcommands that `tikv-ctl` supports in detail. Some
 
 Use the `raft` subcommand to view the status of the Raft state machine at a specific moment. The status information includes two parts: three structs (**RegionLocalState**, **RaftLocalState**, and **RegionApplyState**) and the corresponding Entries of a certain piece of log.
 
-Use the `region` and `log` subcommands to obtain the above information respectively. The two subcommands both support the remote mode and the local mode at the same time. Their usage and output are as follows:
+Use the `region` and `log` subcommands to obtain the above information respectively. The two subcommands both support the remote mode and the local mode at the same time.
+
+For the `region` subcommand:
+
+- To specify the Regions to be viewed, use the `-r` option. Multiple Regions are separated by `,`. You can also use the `--all-regions` option to view all Regions (Note that `-r` and `--all-regions` cannot be used at the same time).
+- To limit the number of Regions to be printed, use the `--limit` option (default: `16`).
+- To query which Regions are included in a certain key range, use the `--start` and `--end` options (default: no range limit, in Hex format).
+
+To print the Region with id `1239`, use the following command:
 
 ```bash
```

Comment:
I think we can unify this file ` ```bash ` to the ` ```shell`🤔
If you don't think it's necessary, we can just change this pr to `  ```shell `

Reply:
https://github.com/pingcap/docs/pull/11526/commits/8817dae39823673cff2ed9364acd8808ea220549 PTAL

Reply:
BTW, can you remove all `$ ` to unify? I will do the same in docs-cn.